etcd/containers/0.12.0/etcd/run.sh

```diff
@@ -0,0 +1,351 @@
+#!/bin/bash
+if [ "$RANCHER_DEBUG" == "true" ]; then set -x; fi
+
+# loop until metadata wakes up...
+STACK_NAME=$(wget -q -O - ${META_URL}/self/stack/name)
```

Comment:
`META_URL` needs to be defined above this line, but instead it is found in line 13.